src/less/components.less

```diff
@@ -48,14 +48,14 @@
 @import 'components/reveals';
 @import 'components/search';
 @import 'components/selectors';
-@import 'components/stepper'; // BETA
+@import 'components/accordion/stepper'; // BETA
 @import 'components/tables';
 @import 'components/tabs';
-@import 'components/text-input';
-@import 'components/textarea';
+@import 'components/text-input/index';
+@import 'components/textarea/index';
 @import 'components/toasts';
 @import 'components/tooltips';
 @import 'components/typography/index';
 
 // DEPRECATED
-@import 'components/hxTextCtrl'; // remove in v1.0.0
+@import 'components/form/_hxTextCtrl-DEPRECATED'; // remove in v1.0.0
```

Comment:
Can we move the import for this file to the bottom of `src/less/components/form/index.less`?

Reply:
Moved from `src/less/components.less` -> `src/less/components/form/index.less`:

```less
// DEPRECATED
@import 'components/form/_hxTextCtrl-DEPRECATED'; // remove in v1.0.0
```